packages/grafana-data/src/types/datasource.ts

```diff
@@ -554,6 +554,7 @@ export interface DataQueryRequest<TQuery extends DataQuery = DataQuery> {
   rangeRaw?: RawTimeRange;
   timeInfo?: string; // The query time description (blue text in the upper right)
   panelId?: number;
+  panelPluginId?: string;
```

Comment:
I think `panelPluginType` would be a better name since `panelId` exists on the line above. It didn't click for me that it contains the type of the panel issuing the query.

Reply:
Yeah, fair enough, don't have a strong opionion her. Will update soon.